pom.xml

```diff
@@ -180,6 +180,12 @@
       <scope>provided</scope>
     </dependency>
 
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
```

Comment:
Since JDK9, javax annotaiton (e.g., `@Generated`) needs to be explicitly imported